src/foam/dao/java/JDAO.js

```diff
@@ -32,11 +37,20 @@ foam.CLASS({
               .setCreateFile(true)
               .build());
 
-            // create a composite journal of repo journal
-            // and runtime journal and load them all
+            /* Create a composite journal of repo journal and runtime journal
+              and load them all.*/
+            X initialStorageX = x;
+            try {
+              String initialStorageSource = System.getProperty("INITIAL_JOURNAL_SOURCE");
+              initialStorageX = x.put(foam.nanos.fs.Storage.class,
+                  new Storage(initialStorageSource));
```

Comment:
What is the format we need for INITIAL_JOURNAL_SOURCE to point to a resource in a JAR?

Reply:
"resource"?

it results in a classLoader call so it's not a complete path. 
path part would be "journals/name.0"

Reply:
Using a classLoader in Storage now.